main.go

```diff
@@ -47,7 +67,7 @@ func main() {
 		panic(err)
 	}
 
-	time.Sleep(2 * time.Second)
+	time.Sleep(4 * time.Second)
```

Comment:
Just a Nit: Is there a specific reason we are waiting 10/4/2 seconds? Sounds rather random.